docs/admin/service-accounts-admin.md

```diff
@@ -96,14 +96,15 @@ account, and the controller will update it with a generated token:
 ```json
 secret.json:
 {
-	"kind": "Secret",
-	"metadata": {
-		"name": "mysecretname",
-		"annotations": {
-			"kubernetes.io/service-account.name": "myserviceaccount"
-		}
-	}
-	"type": "kubernetes.io/service-account-token"
+    "kind": "Secret",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "mysecretname",
+        "annotations": {
+            "kubernetes.io/service-account.name": "myserviceaccount"
+        }
+    },
+    "type": "kubernetes.io/service-account-token"
```

Comment:
I did try tabs earlier, turns out it didnt fit much.
And now it makes more sense to me after validated by a json validator.
Would you plz kindly have a double check ;-) @liggitt